context_test.go

```diff
@@ -14,3 +14,16 @@ func TestContext(t *testing.T) {
 	assert.Equal(t, ctx.Value("Hello").(string), "world")
 	assert.Equal(t, ctx.Value("Hi").(string), "Goravel")
 }
+
+func TestContextWithCustomKeyType(t *testing.T) {
+	type customKeyType struct{}
+	var customKey customKeyType
+	var customKeyTwo customKeyType
```

Comment:
Please add additional other type keys: int, string, etc.

Reply:
We may merge these two test cases: `TestContext`, `TestContextWithCustomKeyType`.

Reply:
done in https://github.com/goravel/gin/pull/87/commits/3b44987393c32d31ef3c571661b054770f343cdd